.github/workflows/git-artifacts.yml

```diff
@@ -253,9 +253,67 @@ jobs:
         with:
           name: pkg-${{matrix.arch.name}}
           path: artifacts
+  build-arm64:
+    env:
+      MSYSTEM: MINGW64
+      NO_PERL: 1
+      GIT_CONFIG_PARAMETERS: "'user.name=CI' 'user.email=ci@git'"
+    runs-on: windows-latest
+    steps:
+    - uses: actions/checkout@v1
+    - name: download git-sdk-64-minimal
```

Comment:
I copied this from the regular CI file. Would we be able to use the cached `git-sdk-64-build-installers` here as well? If yes, I'm happy to change it accordingly 👍🏼

Reply:
Yes, `build-installers` should include all that `makepkg-git` includes, and `makepkg-git` should include all that `minimal` includes.